File: 백준/Silver/1978. 소수 찾기/소수 찾기.py
N = int(input())
if(N<=100):
    arr = list(map(int,input().split(" ",N)))
    count = 0
    for i in range(N):
        j = 2
        if(arr[i]==2):
            count+=1
        while(j<arr[i]):
            if(arr[i]%j==0):
                break
            j+=1
            if(j==arr[i]):
                count+=1
print(count)

File: 백준/Silver/1978. 소수 찾기/소수 찾기.c
#include<stdio.h>
#include<stdlib.h>

int main() {
	int N;
	scanf("%d", &N);
	if (N <= 100) {
		int* arr = malloc(sizeof(int) * N);
		for (int i = 0; i < N; i++) {
			scanf("%d", &arr[i]);
		}
		int count=0;
		for (int i = 0; i < N; i++) {
			int j = 2;
			if (arr[i] == 2) {
				count++;
			}
			while (j < arr[i]) {
				if (arr[i] % j == 0) {
					break;
				}
				j++;
				if (j == arr[i]) {
					count++;
				}
			}
		}
		printf("%d", count);
		free(arr);
	}
	return 0;
}